src/Symfony/Component/Validator/composer.json

```diff
@@ -41,10 +41,11 @@
         "symfony/property-info": "^5.3",
         "symfony/translation": "^4.4|^5.0",
         "doctrine/annotations": "^1.10.4",
-        "doctrine/cache": "~1.0",
+        "doctrine/cache": "^1.11",
```

Comment:
Same as above with 2.0

Reply:
This needs more changes in the Validator component. Under certain circumstances, we still use an array cache from Doctrine. I'll do this in a follow-up.

Reply:
Sounds good, thank you!